src/utilities/bcl/BCLMeasure.cpp

```diff
@@ -1074,9 +1074,13 @@ bool BCLMeasure::checkForUpdatesFiles() {
     setMeasureLanguage(MeasureLanguage::Ruby);
     m_bclXML.removeFile(m_directory / "measure.py");
   } else if (hasRuby) {
-    setMeasureLanguage(MeasureLanguage::Ruby);
+    if (measureLanguage() != MeasureLanguage::Ruby) {
+      setMeasureLanguage(MeasureLanguage::Ruby);
+    }
   } else if (hasPython) {
-    setMeasureLanguage(MeasureLanguage::Python);
+    if (measureLanguage() != MeasureLanguage::Python) {
+      setMeasureLanguage(MeasureLanguage::Python);
+    }
```

Comment:
Fixup  failing BCLMeasure test. When the measureLanguage is already correct **including when it's not set and defaulted to `Ruby`**, don't set it in the XML or it'll trigger a new versionId.

@kbenne I expect this will be the last change needed to get CI all green